### ISSUE TITLE
Python Linter

### ISSUE THREAD

**Issue by aaron97neu** [oats-center/isoblue-avena#102]
We are starting to write a large amount of python using varying styles. We should start using a linter to enforce formatting and stylistic guidelines

[Pylint](https://pylint.org/) seems be the defacto linter. By default it enforces PEP8 which is the official standard. I don't necessarily agree with all of PEP8's standards and we can customize it with a `pylintrc`, however we should keep that to a minimum 

Is it possible to include this with our CI stuff as well? That might be cool

**Comment by abalmos** [comment]
Agreed. I personally prefer to use auto formaters (autopep8 in this case) and just stop having format arguments. Lint rules would be a great add. We can create tasks to run pylint for sure.

**Comment by facastiblancor** [comment]
I agree too. That way we have consistency across all scripts. I am also in favor of lint rules.
